src/globals/scss/_colours.scss

```diff
@@ -56,6 +56,7 @@ $govuk-grey-2: #bfc1c3;
 $govuk-grey-3: #dee0e2;
 $govuk-grey-4: #f8f8f8;
 $govuk-white: #ffffff;
+$govuk-black: #000000;
```

Comment:
Is this intentionally different to `#0b0c0c` https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/colours/_palette.scss#L50